packages/dotcom-ui-header/src/interfaces.d.ts

```diff
@@ -17,4 +17,4 @@ export type THeaderProps = THeaderOptions & {
   data: TNavigationData
 }
 
-export type THeaderVariant = 'simple' | 'large-logo'
+export type THeaderVariant = 'simple' | 'large-logo' | 'sticky'
```

Comment:
**Reviewer note:** The `sticky` variant allows us to easily reuse the same top right components for the my account / sign in and subscribe links in the sticky header as the main header but whilst also varying the `tabIndex` attribute in order to remove the link from the tab order when navigating by keyboards